asllib/doc/asl.spec

```diff
@@ -1796,6 +1803,14 @@ relation eval_catchers(env: envs, catchers: list0(catcher), otherwise_opt: optio
   math_layout = [_,_],
  };
 
+semantics function find_catcher(tenv: static_envs, v_ty: ty, catchers: list0(catcher)) ->
+  (catcher_opt: option(catcher))
+{
+  "returns the first catcher clause in {catchers} that matches the type {v_ty} in {catcher_opt} when one exists, and $\None$ otherwise.
```

Comment:
```suggestion
  "returns the first catcher clause in {catchers} that matches the type {v_ty} in {catcher_opt} if one exists.
  Otherwise, it returns $\None$.
```